cmd/kusk/cmd/uninstall.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	kuskv1 "github.com/kubeshop/kusk-gateway/api/v1alpha1"
+	"github.com/kubeshop/kusk-gateway/cmd/kusk/internal/errors"
+	"github.com/kubeshop/kusk-gateway/cmd/kusk/internal/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var confirm bool
+
+var uninstallCmd = &cobra.Command{
+	Use:   "uninstall",
+	Short: "Uninstall kusk-gateway, envoy-fleet, api, and dashboard in a single command",
+	Long: `
+	Uninstall kusk-gateway, envoy-fleet, api, and dashboard in a single command.
+
+	$ kusk uninstall
+
+	Will install kusk-gateway, a public (for your APIS) and private (for the kusk dashboard and api)
+	envoy-fleet, api, and dashboard in the kusk-system namespace using helm.
+
+	$ kusk install --name=my-release --namespace=my-namespace
```

Comment:
```suggestion
	$ kusk uninstall --name=my-release --namespace=my-namespace
```

Reply:
This is not applicable anymore i will have it removed. Just a copy paste garbage